Makefile

```diff
@@ -28,6 +28,9 @@ NPROCS ?= 1
 # to half the number of CPU cores.
 GO_TEST_PARALLEL := $(shell echo $$(( $(NPROCS) / 2 )))
 
+# increase test count to reduce chance of missing flaky unit tests
+GO_TEST_FLAGS    ?= -count=5
```

Comment:
We haven't had too much of a problem with flakes on unit tests so far, so I think I would probably leave this off for now. If we do include it we should use `-failfast` as well